frontend/src/test/scala/bloop/nailgun/NailgunTest.scala

```diff
@@ -40,6 +40,7 @@ abstract class NailgunTest {
 
     serverThread.start()
 
+    Thread.sleep(500)
```

Comment:
Leftover from debugging?

Reply:
This is not a leftover from debugging, it's necessary avoid spurious errors in `NailgunTest`.